docs/wiki/[Examples]-Assign-a-Built-in-Policy.md

```diff
@@ -0,0 +1,191 @@
+- Assign the policy definition for `Deploy default Microsoft IaaSAntimalware extension for Windows Server` at the `es_root` Management Group by extending the built-in archetype for `es_root`
+- Assign the policy set definition for `NIST SP 800-53 Rev. 5` at the `es_root` Management Group by extending the built-in archetype for `es_root`
+- Assign the policy definition for `Not allowed resource types` at the `Landing Zones` Management Group by extending the built-in archetype for `es_landing_zones`.
+
+>IMPORTANT: To allow the declaration of custom or expanded templates, you must create a custom library folder within the root module and include the path to this folder using the `library_path` variable within the module configuration. In our example, the directory is `/lib`.
+
+In order to assign built-in policies, there needs to be an assignment file for each policy or policy set definition that we want to use.
+This module already includes assignment files for some built-in policies so it's important to check whether or not one exists before creating your own. You can do this by navigating to  `\modules\archetypes\lib\policy_assignments` and looking for an assignment file that matches the policy you want to assign.
+An example of a built-in policy that already has an assignment file included within the module is the `Kubernetes clusters should be accessible only over HTTPS` policy. The assignment file for this policy is called `policy_assignment_es_deny_http_ingress_aks.tmpl.json`.
+
+As the policies you will use in this example do not already have an assignment file within the module, you will need to create the below files so that you can assign them:
+
+- [lib/policy_assignments/policy_assignment_not_allowed_resource_types.json](#libpolicy_assignmentspolicy_assignment_not_allowed_resource_typesjson)
+- [lib/policy_assignments/policy_assignment_deploy_default_microsoft_IaaSAntimalware_extension_for_windows_server.json](#libpolicy_assignmentspolicy_assignment_deploy_default_microsoft_IaaSAntimalware_extension_for_windows_serverjson)
+- [lib/policy_assignments/policy_assignment_nist_sp_800_53_rev_5.json](#libpolicy_assignmentspolicy_assignment_nist_sp_800_53_rev_5json)
+
+## Create Custom Policy Assignment Files
+
+In order to assign built-in policies or policy sets, you need to create policy assignment files. The first step is to create a `policy_assignments` subdirectory within `/lib`.
+
+>**NOTE**: Creating a `policy_assignments` subdirectory within `\lib` is a recommendation only. If you prefer not to create one or to call it something else, the custom policies will still work.
+
+You will then need to create a file named `policy_assignment_not_allowed_resource_types.json` within the `policy_assignments` directory. Copy the below code in to the file and save it.
```

Comment:
add another note saying the file name is not important, but the key in the JSON file is. As per the https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/wiki/%5BUser-Guide%5D-Archetype-Definitions doc